Update query-less to queryless (#348)
grafana/traces-drilldown: CONTRIBUTING.md

# Grafana Traces Drilldown

> Grafana app plugin that provides a queryless way for users to navigate and visualize trace data stored in Tempo.

Reviewer: Slight restructure to make it a complete sentence and to remove passive voice. 
```suggestion
The Grafana Traces Drilldown app plugin lets users navigate and visualize trace data stored in Tempo without complex queries.
```

> The Grafana Traces Drilldown app lets users navigate and visualize trace data stored in Tempo without complex queries.

Author: ```suggestion
Use Rate, Errors, and Duration (RED) metrics derived from traces to investigate and understand errors and latency issues within complex distributed systems.
```

> Use Rate, Errors, and Duration (RED) metrics derived from traces to investigate and understand errors and latency issues within complex distributed systems.

Author: ```suggestion
The Grafana Traces Drilldown app lets users navigate and visualize trace data stored in Tempo without complex queries.
```
